cli.rs

```diff
@@ -203,12 +221,9 @@ fn run_honggfuzz(target: &str, timeout: Option<i32>) -> Result<(), Error> {
             fuzzer, target
         ))?;
 
-    ensure!(
-        fuzzer_bin.success(),
-        "{} quit with code {}",
-        fuzzer,
-        fuzzer_bin
-    );
+    if !fuzzer_bin.success() {
+        Err(FuzzerQuit)?;
```

Comment:
I'm very intrigued by this line.
Do people often use this notation instead of `return Err(FuzzerQuit);` ?

Reply:
It's not the same—this converts the error!

Reply:
ah yes I forgot that, but then why not
```rust
return Err(FuzzerQuit).into();
```
I see that this is more verbose but I got used to having a mental representation of `?` as a way to say "we are not interested in explicitly handling the error here but we want the use the result". 
And here, we are explicitly creating this error and we are not interested in the non-existent `Ok()` case.

I understand this is more coherent in the sense that with this, all errors come from "?" so this is easy to skim the code and see those.

Reply:
Anyway this is an interesting notation that I never saw before. I just need some time to wrap my head around it.

Reply:
Interesting, I don't see `?` as "let's not care about the error here", but more as a "[bubble](https://internals.rust-lang.org/t/bikeshed-a-consise-verb-for-the-operator/7289/29?u=killercup) up the error".